File: RepoTransBench/target_projects/Python/Go/mattupstate_overholt/tests/original/tasks_test.go
package original

import (
	"strings"
	"testing"
	"mattupstate_overholt/overholt/tasks"
	"bytes"
	"fmt"
	"io"
	"os"
)

func captureStdout(f func()) string {
	var buf bytes.Buffer
	stdout := os.Stdout
	r, w, _ := os.Pipe()
	os.Stdout = w

	outC := make(chan string)
	go func() {
		var b [1024]byte
		n, _ := r.Read(b[:])
		outC <- string(b[:n])
	}()

	f()
	w.Close()
	os.Stdout = stdout
	out := <-outC
	return out
}

func TestSendManagerAddedEmailPrints(t *testing.T) {
	output := captureStdout(func() {
		tasks.SendManagerAddedEmail("user1@example.com", "user2@example.com")
	})
	if !strings.Contains(output, "sending manager added email") {
		t.Errorf("Expected output to contain 'sending manager added email', got '%s'", output)
	}
}

func TestSendManagerRemovedEmailPrints(t *testing.T) {
	output := captureStdout(func() {
		tasks.SendManagerRemovedEmail("user3@example.com")
	})
	if !strings.Contains(output, "sending manager removed email") {
		t.Errorf("Expected output to contain 'sending manager removed email', got '%s'", output)
	}
}

File: RepoTransBench/target_projects/Python/Java/mattupstate_overholt/src/test/java/com/overholt/original/TasksTest.java
package com.overholt.original;

import com.overholt.tasks.Tasks;
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

import java.io.ByteArrayOutputStream;
import java.io.PrintStream;

class TasksTest {

    @Test
    void testSendManagerAddedEmailPrints() {
        ByteArrayOutputStream outContent = new ByteArrayOutputStream();
        PrintStream originalOut = System.out;
        System.setOut(new PrintStream(outContent));
        try {
            Tasks.sendManagerAddedEmail("user1@example.com", "user2@example.com");
            System.out.flush();
            String output = outContent.toString();
            assertTrue(output.contains("sending manager added email"));
        } finally {
            System.setOut(originalOut);
        }
    }

    @Test
    void testSendManagerRemovedEmailPrints() {
        ByteArrayOutputStream outContent = new ByteArrayOutputStream();
        PrintStream originalOut = System.out;
        System.setOut(new PrintStream(outContent));
        try {
            Tasks.sendManagerRemovedEmail("user3@example.com");
            System.out.flush();
            String output = outContent.toString();
            assertTrue(output.contains("sending manager removed email"));
        } finally {
            System.setOut(originalOut);
        }
    }
}